readme.txt

```diff
@@ -135,33 +175,7 @@ Please note that our support for the checkout block is still experimental and th
 
 = 5.4.0 - 2023-02-01 =
 * Add - Add logging and order notes when WCPay Subscriptions are suspended or put on-hold.
-* Add - Highlight subscriptions with overdue payment in list view with red icon == Changelog ==
```

Comment:
@jbordonado weird changes in `readme.txt` but I see that it syncs with `changelog.txt`.

Reply:
That's weird, I fixed the 5.4.0 weirdness in https://github.com/Automattic/woocommerce-payments/pull/5681 last week.

Reply:
When I checkout the branch locally, I don't see these changes. Can you confirm?

Reply:
That explains it!